sections/faqs/reverse-selectors.js

```diff
@@ -51,6 +51,25 @@ const ReverseSelectors = () => md`
 
   We could have nested the color-changing rule within our Link component, but then we'd have to
   consider both sets of rules to understand why Icon behaves as it does.
-`
 
-export default ReverseSelectors
+  ### Caveat
```

Comment:
I'm not sure whether we'll need to add another heading here 🤔

Reply:
It's a good heading Phil.

More seriously, where we're receiving bug reports about misconceptions around this behaviour I'm inclined to be emphatic.

Reply:
Alright :+1: